### ISSUE TITLE
  tlmgr: Local TeX Live (2021) is older than remote repository (2022).

### ISSUE THREAD

**Issue by erikw** [yegor256/latexmk-action#2]
First, thanks for this great action. I use it a lot on a daily basis!

Today I started to get this build error in one of my repos using this action:

```
  tlmgr: Local TeX Live (2021) is older than remote repository (2022).
  Cross release updates are only supported with
    update-tlmgr-latest(.sh/.exe) --update
  See https://tug.org/texlive/upgrade.html for details.
  The command '/bin/sh -c tlmgr --verify-repo=none update --self' returned a non-zero code: 1
  
  Warning: Docker build failed with exit code 1, back off 3.531 seconds before retry.
```

This should originate from this line in the `Dockerfile`:
https://github.com/yegor256/latexmk-action/blob/102bca8b56592dd12e284fda30da296eb07d2d72/Dockerfile#L33

Here's an example of a build in a public repo that fails:
https://github.com/erikw/resume/runs/5828710504?check_suite_focus=true

How could we update Tex Live to the 2020 version?

**Comment by yegor256** [comment]
@erikw I just released version 0.4.0, which has TeXlive 2022. Please, try this new version.

**Comment by erikw** [comment]
@yegor256 I can confirm that `0.4.0` fixes the reported issue. Thanks!
